lib/ui/page/home/page/chat/message_field/controller.dart

```diff
@@ -209,6 +209,8 @@ class MessageFieldController extends GetxController {
   /// registered.
   bool _handlersRegistered = false;
 
+  final ScrollController attachmentsScrollController = ScrollController();
```

Comment:
https://github.com/team113/messenger/blob/main/CONTRIBUTING.md#documentation-1
https://github.com/team113/messenger/blob/main/CONTRIBUTING.md#classes-constructors-fields-and-methods-ordering